### ISSUE TITLE
markdown export of a book results in page-title headers being directly concatenated to the previous page's text

### ISSUE THREAD

**Issue by flan** [BookStackApp/BookStack#3341]
### Describe the Bug

Observed in BookStack 22.02.3

After creating a book comprised of multiple pages, I went to export it as markdown for use in another rendering pipeline. This mostly worked as expected, but there was an oddity: where one page ends and the next begins, no linebreaks were inserted, so the output looks like this:

```
nothing is allowed to fall through the cracks.# Discovery
```

The title of the next page, "Discovery", is rendered as a header, but it is quite literally concatenated to the last sentence of the previous page.

I did not test this against books comprised of chapters.

### Steps to Reproduce

Export, as markdown, any book comprised of pages.

### Expected Behaviour

Rather than what was seen,

```
nothing is allowed to fall through the cracks.# Discovery
```

I expected to see

```
nothing is allowed to fall through the cracks.

# Discovery
```

with two linebreaks at the point of page-concatenation to produce valid markdown.

### Exact BookStack Version

22.02.3

### Hosting Environment

Self-hosted, deployed from installer scripts in a Debian Bullseye VM.

**Comment by ssddanbrown** [comment]
Thanks for reporting @flan, Can confirm this. Have assigned to the next feature release.

**Comment by ssddanbrown** [comment]
This has now been addressed in 1c859e94e03d0407daf55ade43ced245f208f971 and will be part of the next feature release hence will therefore close this.

Thanks again for reporting.
